basketball-api/controllers/GamesController.js

```diff
@@ -14,3 +14,27 @@ exports.getById = async (req, res) => {
     }
     res.send(games)
 }
+
+exports.createNew = async (req, res) => {
+    let game
+    try {
+      game = await Game.create(req.body)
+    } catch (error) {
+      if (error instanceof db.Sequelize.ValidationError) {
+        res.status(400).send({"error":error.errors.map((item)=> item.message)})
+      } else {
```

Comment:
Test for foreign key errors and inform the user if such exists.
[example](https://github.com/kivikangur-tthk/games-api-TARgv21/blob/main/controllers/GamePlaysController.js)

Reply:
Tegin pivotis taski ja homme vaatan üle millised tabelid peaks seda kasutama ja millised mitte.